auth_proxy/models/oauth.py

```diff
@@ -68,7 +70,9 @@ class Grant(Base):
     _scopes = Column('scopes', Text)
 
     def delete(self):
-        self.expires = 0
+        self.expires = datetime.now()
```

Comment:
This is the actual fix I introduced in https://github.com/sync-for-science/auth-proxy/pull/17, but it doesn't require that the OAuthService attach the db session to this class. It's still a separation of concerns violation, but it's isolated to one class instead of two.